version.yaml

```diff
@@ -1,4 +1,4 @@
-current_version: 3.0.0.dev (Teal Blauwbok)
+current_version: 3.0.1.dev (Teal Blauwbok)
```

Comment:
@spirillen it's generated with `PyFunceble --prod` (`--production`)

Reply:
So it's important to work from an isolated environment 😉

Reply:
So you are asking me to undo changes in this particular file?